app/javascript/packages/document-capture/components/unknown-error.tsx

```diff
@@ -34,7 +34,11 @@ function UnknownError({
     );
   }
   if (err) {
-    return <p key={err.message}>{err.message}</p>;
+    return (
+      <p className="display-inline" key={err.message}>
```

Comment:
Why was it required to make the paragraph inline?

Reply:
I wanted to display the help center link at the end of the paragraph on the "Review your images and try again" page.  When I added the link in lines 36-41 of `UnknownError`, the link would also display on the "We could not verify your ID" page, which was not the desired outcome.  I added the link in `DocumentCaptureReviewIssues` instead and then made this paragraph inline, so that the link wouldn't display on a separate line than the paragraph.

Is there another approach you think would be better?  I'm curious to hear other perspectives.

Reply:
I think it'd be nice if the link could be a child of the paragraph, so that it's semantically part of the same block of text, not relying on CSS to collapse the two.

Maybe either... ?
* Move the `<p>` out of `UnknownError` and instead render it at the place the component is used
   * e.g. `<p><UnknownError {/* ... */} /><a {/* ... */} /></p>`
* Or, somehow incorporate the link into the `err.message` itself, perhaps through logic of `serivces/upload.ts` and specifically the `UploadFormEntryError` class implemented there.

Reply:
I prefer the second option, but I'm unsure how to implement that in a resilient way.  

I can add a field, `helpCenterLink` to `UploadFormEntryError` in `services/upload.ts`.  The problem is when I try to use the `getHelpCenterURL` helper method in `upload.ts`.  I get a console error saying that I'm breaking the Rules of Hooks, I think because I called `useEffect` outside of a function component.  

Are you also against hardcoding the help center url in `UploadFormEntryError` in `upload.ts`?

Reply:
Yeah I don't think using React-isms will work very well in `upload.ts`, since that file is mostly plain-ol' JavaScript.

I actually think the approach you've landed in `UnknownError` could work well, and maybe we just get the help center link in that component? i.e. move what you have in `getHelpCenterLink` into `UnknownError`.

Reply:
> I actually think the approach you've landed in UnknownError could work well, and maybe we just get the help center link in that component? i.e. move what you have in getHelpCenterLink into UnknownError.

👍🏻  I'm going for that approach.   I added `hasDismissed` as a prop to `UnknownError` and used conditional logic to keep the help center link from displaying on the "We could not verify your ID" page.